pages/Berries/jaboca.md

```diff
@@ -0,0 +1,49 @@
+# Jaboca
+
+Jaboca is a type of [Berry](https://wiki.pokeclicker.com/) that can be planted in the [Farm](https://wiki.pokeclicker.com/). It can be obtained by having at least one empty plot and at least 600 points of bitter on the field. The [Durin](https://wiki.pokeclicker.com/) must be unlocked to unlock this berry. Various [Farming Setups](https://wiki.pokeclicker.com/) using Jaboca can be found [here](https://wiki.pokeclicker.com/).
+
+ ## Table of Contents
+ 1. [Growth Tables](#growth_tables)
+    1. [Growth Stages](#growth_stages)
+    2. [Ripening Time with Modifiers](#ripening)
+ 2. [Flow Chart](#flow_chart)
+ 3. [Mutation Tables](#mutation)
+    1. [Possible Planting Formations](#planting)
+ 
+### Growth Tables <a name="growth_tables"></a>
+
+#### Growth Stages
+
+|                                   |     **Sprout**   |  **Taller**     | **Bloom**       | **Berry**                          |
+|:---------------------------------:|:----------------:|:---------------:|:---------------:| :---------------------------------:|
+| **Time passed since planting**    |  1 hour 12 mins  | 2 hours 12 mins | 4 hours 36 mins | 9 hours 18 mins - 18 hours 36 mins |
+| **Animation**                     |    -             |   -             | -               | -                                  |
+| **Aura (Roaming Pok�mon Chance)** |                  |  x1.005         | x1.01           | x1.015                             |
+
+### Ripening Time with Modifiers
+
+| **Modifier**           |     **Ripe-Wither Time**           |  **Boost Mulch Needed to Ripen** |
+|:----------------------:|:----------------------------------:|:--------------------------------------:|
+| **None**               | 9 hours 18 mins - 18 hours 36 mins | ---                                    | 
+| **Mulch or Sprayduck** | 6 hours 12 mins - 12 hours 24 mins |   75 mulch                             |
+| **Mulch & Sprayduck**  | 4 hours 8 mins - 8 hours 16 mins   |  50 mulch                              |
+
+## Flow Chart
+
+<!--- Comment? How should we do this? -->
```

Comment:
At the moment we don't have a plugin for flow charts, will look into one, but otherwise probably just have to upload & link an image.